examples/testapp/config/headers.js

```
@@ -0,0 +1,17 @@
+function getCrossOriginOpenerPolicyHeaders() {
+    return {
```

ESLint

[error] Delete `··`

(prettier/prettier)